### PR TITLE
HV-528 Add IndexOf to Observable Collection

### PR DESCRIPTION
**Pull request by bencampbell-wf**
# Problem
There is no way to test if an object is a member of a collection or it's position in the list.

# Solution
Expose indexOf of the underlying array through the public interface.

## Dependency
* [HV-99 Depends on HV-528](https://github.com/Workiva/wf-js-annotations/pull/549)

# Testing
* Added Unit Tests 
* All Unit Tests pass

### CODE REVIEW

**Comment by rmconsole-wf** [comment]
When this pull is merged I will use the following information:
		Version: wf-common 1.0.1
		Release Ticket: HY-1898

***
*Last updated on Wednesday, April 01 03:26 PM CST*

File: test/observable/CollectionSpec.js [hunk 1]
```diff
@@ -106,6 +106,21 @@ define([
                 expect(collection._collection).toEqual(['a value','c value']);
             });
 
+            it('should return index of given value', function() {
+                collection.append('a value');
+                collection.append('b value');
+
+                expect(collection.indexOf('a value')).toEqual(0);
+
```
Comment (tomconnell-wf) [inline test/observable/CollectionSpec.js]: This is an extra line that could be removed.

**Comment by tomconnell-wf** [comment]
+1 - Crushed it.  I will replus after you fix the extra whitespace.
grape

**Comment by tomconnell-wf** [comment]
+1

**Comment by todbachman-wf** [comment]
+10

**Comment by charliekump-wf** [comment]
QA Resource Approval: +1

- [x] Testing instruction
- [x] Dev +1's
- [x] Dev/QA +10 with detail of what was tested
- [x] Unit test created/updated 
- [x] All unit tests pass 

Merging into master.

**Pull request merged**